### DISCUSSION TITLE
Пропавшие иконки в дискорде

### DISCUSSION THREAD

**Discussion by SirWaltray** [bol-van/zapret#841] [Q&A]
В последние дни пропали все аватарки людей, все эмодзи и все обложки каналов. Есть ли возможность починить всё это?

**Comment by HentayHentay** [comment]
Проблемы в блокировке некоторых забугорных CDN.
Требуется перебросить трафик за пределы контроля своего провайдера

**Comment by VatruhinReal** [comment]
> Проблемы в блокировке некоторых забугорных CDN. Требуется перебросить трафик за пределы контроля своего провайдера

И что это значит? ВПН типа только спасет? Никак не пофиксить чтоли?

**Comment by HentayHentay** [comment]
> > Проблемы в блокировке некоторых забугорных CDN. Требуется перебросить трафик за пределы контроля своего провайдера
> 
> И что это значит? ВПН типа только спасет? Никак не пофиксить чтоли?

Можешь попробовать докинуть хосты, откуда грузиться иконки..(гуглим)
Ну и в случае неудачи дискорд в туннель

**Comment by BiNoopsGITHUB** [comment]
> > > Проблемы в блокировке некоторых забугорных CDN. Требуется перебросить трафик за пределы контроля своего провайдера
> > 
> > 
> > И что это значит? ВПН типа только спасет? Никак не пофиксить чтоли?
> 
> Можешь попробовать докинуть хосты, откуда грузиться иконки..(гуглим) Ну и в случае неудачи дискорд в туннель

в hosts такой формат "127.0.0.1       site.com", что нужно вписать для иконок дискорда?

**Comment by VatruhinReal** [comment]
[list-general.txt](https://github.com/user-attachments/files/17714020/list-general.txt)

Замените старый файлик на новый, переустановив сервис (если у вас службой)

**Reply by CHUPAKABRA8** [reply]
Спасибо большое!

**Comment by c4kef** [comment]
> [list-general.txt](https://github.com/user-attachments/files/17714020/list-general.txt)
> 
> Замените старый файлик на новый, переустановив сервис (если у вас службой)

спасибо!
